app/dashboard/views.py

```diff
@@ -426,6 +426,13 @@ def process_bounty(request):
 
     return TemplateResponse(request, 'process_bounty.html', params)
 
+def onboard(request):
+    "FTUX flow"
```

Comment:
You'll want to wrap this in triple double quotes. Something like: `"""Handle FTUX entry for onboard flow."""`